src/docs/product/sentry-basics/guides/grouping-and-fingerprints/index.mdx

```diff
@@ -36,7 +36,9 @@ Fortunately you can change the default grouping behavior to make certain issue t
 
 # Solutions
 
-There are three different approaches for updating how events group into issues. The first approach is for merging together historical issues already created. We'll call this **Merging Similar Issues**. The second is for setting rules so next incoming issues will get grouped together. We'll call this **Fingerprint Rules**. The third is **SDK Side Fingerprinting**. The difference between SDK side and Server side is the data elements on the exception and stack traces which you can use for matching issues.
+The easiest way to improve grouping is to [upgrade the project's grouping algorithm](/product/data-management-settings/event-grouping#grouping-algorithms).
+
+Beside upgrading, there are three different approaches for updating how events group into issues. The first approach is for merging together historical issues already created. We'll call this **Merging Similar Issues**. The second is for setting rules so next incoming issues will get grouped together. We'll call this **Fingerprint Rules**. The third is **SDK Side Fingerprinting**. The difference between SDK side and Server side is the data elements on the exception and stack traces which you can use for matching issues.
```

Comment:
This paragraph seems to repeat most of what's already said about merging and fingerprint rules. Suggest just taking what you've written here about SDK side fingerprinting and adding it before the line that says "More on SDK Side Fingerprinting here." 

```suggestion
```

Reply:
Decided not to touch this file at all for now (except for fixing the headings).